app/assets/stylesheets/external-links.scss

```diff
@@ -0,0 +1,18 @@
+
+.external-links {
+
+  .media {
+    .icon {
+      fill: #4a4a4a;
+    }
+  }
+
+  li {
+    display: flex;
+    align-items: flex-end;
```

Comment:
Sorry to make you change this one again!

I just realized that on mobile it looks kinda broken:

* `align-items: flex-end`:
  <img width="374" alt="Screen Shot 2021-01-05 at 18 41 55" src="https://user-images.githubusercontent.com/67437/103679942-c89e6300-4f85-11eb-9869-4a3b97ed3ae7.png">
* `align-items: center`:
<img width="378" alt="Screen Shot 2021-01-05 at 18 42 02" src="https://user-images.githubusercontent.com/67437/103679949-cb00bd00-4f85-11eb-9b82-ab1c26baa764.png">
* `align-items: flex-start`:
<img width="377" alt="Screen Shot 2021-01-05 at 18 42 11" src="https://user-images.githubusercontent.com/67437/103679953-cc31ea00-4f85-11eb-99f2-7021bc30a636.png">


I'd say `align-items: flex-start` looks the best.

Reply:
Good catch, thanks! Agree with you that `flex-start` looks the best.